### PR TITLE
DOC : Correcting bug on Documentation Page (Byteswapping)

### PR DESCRIPTION
**Pull request by vrindaaa**
Issue #9535
Correcting bug on the documentation page : Byteswapping.

### CODE REVIEW

File: numpy/doc/byteswapping.py [hunk 1]
```diff
@@ -31,16 +31,16 @@
 3 + 2, the 4 bytes in memory would contain respectively: 0, 1, 3, 2.
 The bytes I have loaded from the file would have these contents:
 
->>> big_end_str = chr(0) + chr(1) + chr(3) + chr(2)
->>> big_end_str
-'\\x00\\x01\\x03\\x02'
+>>> big_end_array = bytearray([0,1,3,2])
+>>> big_end_array
+bytearray(b'\\x00\\x01\\x03\\x02')
```
Comment (eric-wieser) [inline numpy/doc/byteswapping.py]: Calling this `_array` is definitely better than `_str` was, but it's a little similar to the `_arr` used below. Can you use `_buffer` or `_bytes` instead?

**Review by eric-wieser — APPROVED** [review]

**Comment by eric-wieser** [comment]
For next time: you should squash your commits if one is a correction to the other like it is here, and you should start the commit message with DOC.

We can fix both those things when merging though, so not a big deal

**Comment by vrindaaa** [comment]
@eric-wieser okay I will keep that in mind next time

**Comment by charris** [comment]
Thanks @vrindaaa .

**Pull request merged**